modules/apps/transfer/keeper/migrations_test.go

```diff
@@ -485,3 +485,51 @@ func (suite *KeeperTestSuite) TestMigratorMigrateMetadata() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestMigratorTotalEscrowsIntProtoToInt() {
+	var expectedTotalEscrowForDenoms []sdk.Coin
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+	}{
+		{
+			"success",
+			func() {
+				// set total escrow for denoms amounts in store using legacy IntProto encoding
+				store := suite.chainA.GetSimApp().TransferKeeper.KVStoreService.OpenKVStore(suite.chainA.GetContext())
+				key := transfertypes.TotalEscrowForDenomKey("ibc/EB7094899ACFB7A6F2A67DB084DEE2E9A83DEFAA5DEF92D9A9814FFD9FF673FA")
```

Comment:
Should test with more key entries than this one. This was just to get the test up and running and prove the migration was successful. 

I can try to add some more cases :)